practice_07.13.2019_events/Task3_IDisposable/FileSystem.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+
+namespace practice_07._13._2019_events.Task3_IDisposable
+{
+    public class FileSystem : IDisposable
```

Comment:
Please look through documentation about dispose pattern:
https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose

Reply:
New DisposeClass in Task3 was created and pattern was added. I hope I understood everything correctly. Fixed